cognite/neat/graph/extractors/_iodd.py

```diff
@@ -31,20 +33,31 @@ class IODDExtractor(BaseExtractor):
         root: XML root element of IODD XML file.
         namespace: Optional custom namespace to use for extracted triples that define data
                     model instances. Defaults to DEFAULT_NAMESPACE.
+        device_tag: Optional user specified unique tag for actual equipment instance. If not provided, a randomly
+        generated UUID will be used.
     """
 
     device_elements_with_text_nodes: ClassVar[list[str]] = ["VendorText", "VendorUrl", "DeviceName", "DeviceFamily"]
+    std_variable_elements_to_extract: ClassVar[list[str]] = ["V_SerialNumber", "V_ApplicationSpecificTag"]
 
-    def __init__(self, root: Element, namespace: Namespace | None = None):
+    def __init__(self, root: Element, namespace: Namespace | None = None, device_tag: str | None = None):
```

Comment:
Strong suggestion rename `device_tag` to `device_id` as in the rest of the code you are using `device_id`, also easier to follow the code.